auth-web/tests/unit/components/ShortNamePaymentHistory.spec.ts

```diff
@@ -17,51 +16,60 @@ sessionStorage.setItem('AUTH_API_CONFIG', JSON.stringify({
 const vuetify = new Vuetify({})
 // Selectors
 const { header, headerTitles, itemRow, itemCell } = baseVdataTable
-const headers = ['Date', 'Description', 'Related Statement Number', 'Amount']
+const headers = ['Date', 'Description', 'Related Statement Number', 'Amount', 'Actions']
 
-describe('ShortNameTransactions.vue', () => {
+describe('ShortNamePaymentHistory.vue', () => {
   setupIntersectionObserverMock()
   let wrapper: Wrapper<any>
   let sandbox: any
   let localVue: VueConstructor<any>
-  let transactionsResponse: any
+  let historyResponse: any
 
   beforeEach(async () => {
     localVue = createLocalVue()
-    transactionsResponse = {
+    historyResponse = {
       items: [
         {
-          'accountBranch': 'Test Branch',
+          'accountBranch': 'Sushi Division',
           'accountId': '3202',
-          'accountName': 'ABC 123',
+          'accountName': 'Odysseus Chiu',
+          'amount': 351.5,
+          'historicalId': 5,
+          'isProcessing': true,
+          'isReversible': false,
+          'shortNameBalance': 368.5,
           'shortNameId': 2,
-          'statementId': 5374449,
-          'transactionAmount': 451.5,
-          'transactionDate': '2023-11-27T16:24:37.522500',
-          'transactionDescription': 'Statement Paid',
-          'transactionId': null
+          'statementNumber': 5374449,
+          'transactionDate': '2024-08-01T00:01:30.885474',
+          'transactionType': 'STATEMENT_REVERSE'
         },
         {
-          'accountBranch': null,
-          'accountId': null,
-          'accountName': null,
+          'accountBranch': 'Sushi Division',
+          'accountId': '3202',
+          'accountName': 'Odysseus Chiu',
+          'amount': 131.5,
+          'historicalId': 3,
+          'isProcessing': false,
+          'isReversible': false,
+          'shortNameBalance': 368.5,
           'shortNameId': 2,
-          'statementId': null,
-          'transactionAmount': 151.5,
-          'transactionDate': '2023-11-24T13:47:47',
-          'transactionDescription': 'Funds Received',
-          'transactionId': 6
+          'statementNumber': 5455966,
+          'transactionDate': '2024-07-31T22:22:31.058547',
+          'transactionType': 'STATEMENT_PAID'
         },
         {
-          'accountBranch': null,
-          'accountId': null,
-          'accountName': null,
+          'accountBranch': 'Sushi Division',
```

Comment:
Where is the Noodle Division?

Reply:
I am leveling that up right now :) one day